src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/InitiateGeneralApplicationHandler.java

```diff
@@ -318,6 +338,13 @@ private CallbackResponse submitApplication(CallbackParams callbackParams) {
                 .build();
             caseData = updatedCaseData;
         }
+        if (featureToggleService.isCoSCEnabled() && isUserLR(caseData)) {
```

Comment:
I believe we can create a new method for this if then we can avoid duplicate code since we've same code at line 273